packages/driver/cross-origin-testing.md

```diff
@@ -0,0 +1,141 @@
+# Cross-origin Testing Technical Overview
+
+The goal of this document is to give a technical overview of the architecture behind the **cy.origin()** command, which enables cross-origin testing in Cypress.
+
+## Definitions
+
+See [Node.js’s URL doc](https://nodejs.org/api/url.html#url-strings-and-url-objects) for a handy breakdown of URL parts
+
+- **domain**: The top-level domain or super-domain of a URL. A hostname without the subdomain. (e.g. `example.com`)
+- **origin**: The combination of the protocol, hostname, and port of a URL. For the purposes of Cypress, the subdomain is irrelevant. (e.g. `http://example.com:3500`)
```

Comment:
```suggestion
- **origin**: The combination of the protocol, hostname, and port of a URL. For the purposes of `cy.origin`, the subdomain is irrelevant. (e.g. `http://example.com:3500`)
```

Reply:
It's true about Cypress in general. Since we set `document.domain`, different subdomains don't cause cross-origin errors, so cy.origin() isn't needed if on the same superdomain.